### PR TITLE
Show the Aleph version when using the -v or --version options

### LINKED ISSUE #3577: BUG: `aleph --version` doesn't output the version
**Describe the bug**

Title

**To Reproduce**

```
$ aleph --version
Python 3.8.10
Flask 2.3.3
Werkzeug 3.0.0
```

**Expected behavior**

It outputs the version

**Aleph version**

3.15.3

### PR DESCRIPTION
**Pull request by maquchizi**
Fixes #3577 

#### Old behaviour
image

#### New behaviour
image

### CODE REVIEW

**Comment by maquchizi** [comment]
Hey @stchris @tillprochaska, please review this tiny PR.

I've (obviously) never contributed to this project before and I wanted to start with something trivial. Let me know if I'm missing anything.

File: aleph/manage.py [hunk 1]
```diff
@@ -82,7 +85,28 @@ def ensure_collection(foreign_id, label):
...
-@click.group(cls=FlaskGroup, create_app=create_app)
+def get_aleph_version(ctx: click.Context, _: click.Parameter, value: Any) -> None:
+    if not value or ctx.resilient_parsing:
+        return
+
+    click.echo(
+        f"Aleph {importlib.metadata.version('aleph')}\n"
+        f"Python {platform.python_version()}\n"
+        f"Flask {importlib.metadata.version('flask')}\n"
+        f"Werkzeug {importlib.metadata.version('werkzeug')}\n",
```
Comment (stchris) [inline aleph/manage.py:96]: I know I'm being really pedantic here, but can you remove the trailing newline?

```suggestion
        f"Werkzeug {importlib.metadata.version('werkzeug')}",
```

File: aleph/manage.py [hunk 2]
```diff
@@ -82,7 +85,28 @@ def ensure_collection(foreign_id, label):
     return collection
 
 
-@click.group(cls=FlaskGroup, create_app=create_app)
+def get_aleph_version(ctx: click.Context, _: click.Parameter, value: Any) -> None:
```
Comment (stchris) [inline aleph/manage.py:88]: A very minor thing, but I find this name better describes what this function does:

```suggestion
def print_aleph_version(ctx: click.Context, _: click.Parameter, value: Any) -> None:
```

**Review by stchris — CHANGES_REQUESTED** [review]
Your contribution is much appreciated, @maquchizi! I left two minor comments, but otherwise this looks good!

File: aleph/manage.py (hunk 1 above)
Comment (maquchizi) [reply aleph/manage.py]: Fair enough. I debated adding that newline in the first place. Fixed in the latest commit.

File: aleph/manage.py (hunk 2 above)
Comment (maquchizi) [reply aleph/manage.py]: Agreed and changed.

Comment (stchris) [reply aleph/manage.py]: Thanks!

File: aleph/manage.py (hunk 1 above)
Comment (stchris) [reply aleph/manage.py]: Great!

**Pull request merged**